### PR TITLE
Docker is 5.6.x centos

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: dockerfiles/centos/is-analytics/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ARG JAVA_HOME=${USER_HOME}/java
+# set wso2 product configurations
+ARG WSO2_SERVER=wso2is-analytics
+ARG WSO2_SERVER_VERSION=5.6.0
+ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+
+RUN yum check-update && \
+    DEBIAN_FRONTEND=noninteractive yum install --downloadonly \
+    #curl \
```
Comment (DilanUA) [inline dockerfiles/centos/is-analytics/Dockerfile]: Shall we remove this commented line from the code?

File: dockerfiles/centos/is-analytics/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# set environment variables
+ENV JAVA_HOME=${JAVA_HOME} \
+    PATH=$JAVA_HOME/bin:$PATH \
+    WSO2_SERVER_HOME=${WSO2_SERVER_HOME} \
+    WORKING_DIRECTORY=${USER_HOME}
+
+# expose ports
+EXPOSE 4000 9764 9444 7712 7612 11225 10006 11001 11501 8082 4041
+# start the WSO2 Carbon server
+
```
Comment (DilanUA) [inline dockerfiles/centos/is-analytics/Dockerfile]: remove additional line

File: dockerfiles/centos/is-analytics/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+WORKDIR ${USER_HOME}
+
+# set environment variables
+ENV JAVA_HOME=${JAVA_HOME} \
+    PATH=$JAVA_HOME/bin:$PATH \
+    WSO2_SERVER_HOME=${WSO2_SERVER_HOME} \
+    WORKING_DIRECTORY=${USER_HOME}
+
+# expose ports
+EXPOSE 4000 9764 9444 7712 7612 11225 10006 11001 11501 8082 4041
```
Comment (DilanUA) [inline dockerfiles/centos/is-analytics/Dockerfile]: Add one blank line after ports exports statement for better clarity.

File: dockerfiles/centos/is-analytics/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Dockerfile for WSO2 Identity Server Analytics #
+This section defines the step-by-step instructions to build the Docker image for WSO2 Identity Server Analytics 5.5.0.
```
Comment (DilanUA) [inline dockerfiles/centos/is-analytics/README.md]: version should be 5.6.0

File: dockerfiles/centos/is/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+WORKDIR ${USER_HOME}
+
+# set environment variables
+ENV JAVA_HOME=${JAVA_HOME} \
+    PATH=$JAVA_HOME/bin:$PATH \
+    WSO2_SERVER_HOME=${WSO2_SERVER_HOME} \
+    WORKING_DIRECTORY=${USER_HOME}
+
+# expose ports
+
```
Comment (DilanUA) [inline dockerfiles/centos/is/Dockerfile]: Shall we remove this extra line?

**Comment by chirangaalwis** [comment]
Note: This PR fixes https://github.com/wso2/docker-is/issues/82.

**Pull request merged**